src/ramses_tx/parsers.py

```diff
@@ -1531,6 +1531,10 @@ def parser_22e9(payload: str, msg: Message) -> Mapping[str, float | None]:
 
 # fan_speed (switch_mode), HVAC
 def parser_22f1(payload: str, msg: Message) -> dict[str, Any]:
+    # Schema B: Vasco and ClimaRad fan remotes send fixed command 06, not 04, like
+    # .I --- 37:117647 32:022222 --:------ 22F1 003 000506 for high
+    # ClimaRad VenturaV1x does not send 22F1 for speed, uses 22F4
+
```

Comment:
Do not have this here - add it to **tests/parsers/code_22F1.log**

NOTE: is is 'scheme' `06`, not schema B

It is best to do this first, and then change the code so that there are no errors when you run:
```
pytest tests/tests/test_parsers.py::test_parsers_from_log_files[code_22f1.log]
```

Reply:
I have added a start for you in the latest push of master:
```
# Scheme 06 - seen with Vasco : 000[234]06 (sent as set of 5)
2024-10-10T18:10:21.527295 073  I --- 29:123150 29:099029 --:------ 22F1 003 000306  #

2024-10-10T22:26:38.421878 041  I --- 29:091138 32:022222 --:------ 22F1 003 000206  #
2024-10-10T23:31:09.943782 042  I --- 29:091138 32:022222 --:------ 22F1 003 000306  #
2024-10-10T23:36:03.885268 042  I --- 29:091138 32:022222 --:------ 22F1 003 000406  #
```

You will need to add what ever is after the `#` before you run **pytest**.

Reply:
Thanks